integration-tests/jail/disk/go_test.sh

```diff
@@ -20,6 +20,8 @@ source /opt/sgxsdk/environment
 source /root/.cargo/env
 echo "sgx mode=" $SGX_MODE
 echo "network id=" $NETWORK_ID
+echo "spid = " $SPID
+echo "ias_api_key = " $IAS_API_KEY 
```

Comment:
these values are typically set as secrets on CI, so do not to print them

Reply:
ok